modules/storages/app/components/storages/admin/forms/oauth_client_form_component.rb

```diff
@@ -33,16 +33,17 @@ class OAuthClientFormComponent < ApplicationComponent
     include OpPrimer::ComponentHelpers
     include OpTurbo::Streamable
 
-    attr_reader :storage
-    alias_method :oauth_client, :model
+    alias_method :storage, :model
 
-    def initialize(oauth_client:, storage:, **)
-      super(oauth_client, **)
-      @storage = storage
-    end
+    options in_wizard: false
 
     def self.wrapper_key = :storage_oauth_client_section
 
+    def form_url
+      query = { continue_wizard: storage.id } if in_wizard
```

Comment:
💚 It as been a while since I saw this. That's a very ruby-esque line, making query nil in case in_wizard is false. =)